packages/forms/src/UIForm/fields/Date/Time.component.js

```diff
@@ -25,14 +25,14 @@ function TimeWidget({
 		onFinish(event, { schema });
 	}
 
-	function onTimeChange(event, { errorMessage: nextErrorMessage, textInput, time }) {
+	function onTimeChange(event, { errorMessage: nextErrorMessage, textInput }) {
 		setState({ errorMessage: nextErrorMessage });
 		const payload = {
 			schema,
 			value: textInput,
 		};
 		onChange(event, payload);
-		if (!nextErrorMessage && time) {
```

Comment:
Simplify the condition, remove redundancy. "no nextErrorMessage" equals to "time is not null".